vcd/datasource_vcd_org_vdc_test.go

```diff
@@ -9,25 +9,24 @@ import (
 	"github.com/hashicorp/terraform/helper/resource"
 )
 
+var vdcName = "TestAccVcdVdcDatasource"
+
 func TestAccVcdVdcDatasource(t *testing.T) {
 	validateConfiguration(t)
 
-	vdcName := TestAccVcdVdc + "ForDataSourceTest"
-
 	var params = StringMap{
 		"ExistingVdcName": testConfig.VCD.Vdc,
 		"VdcName":         vdcName,
 		"OrgName":         testConfig.VCD.Org,
-		"FuncName":        "TestAccVcdVdcDatasource",
+		"FuncName":        vdcName,
 	}
 
+	configText := templateFill(testAccCheckVcdVdcDatasource_basic, params)
```

Comment:
Please change the test in such a way that only one `templateFill` is called. 
This instruction creates the test file in `test-artifacts`. If we call it twice, it will create two files, one of which will contain fields visible to system admin only, and the subsequent binary tests will fail.

Reply:
updated